bazel/linux/qemu.bzl

```diff
@@ -38,33 +47,50 @@ test -n "$QEMU_BINARY" || {{
     exit 10
 }}
 
-if [ -n "$ROOTFS" ]; then
-    QEMU_FLAGS+=("-drive" "file=$ROOTFS,if=virtio,cache=none")
-else
-    QEMU_FLAGS+=("-fsdev" "local,security_model=none,multidevs=remap,id=fsdev-fsRoot,path=/")
-    QEMU_FLAGS+=("-device" "virtio-9p-pci,fsdev=fsdev-fsRoot,mount_tag=/dev/root")
-    KERNEL_FLAGS+=("root=/dev/root" "rootfstype=9p" "init=$INIT")
-    KERNEL_FLAGS+=("rootflags=trans=virtio,version=9p2000.L,msize=5000000,cache=mmap,posixacl")
+if [[ "$ARCH" == "x86" ]]; then
+    if [ -n "$ROOTFS" ]; then
+        QEMU_FLAGS+=("-drive" "file=$ROOTFS,if=virtio,cache=none")
+    else
+        QEMU_FLAGS+=("-fsdev" "local,security_model=none,multidevs=remap,id=fsdev-fsRoot,path=/")
+        QEMU_FLAGS+=("-device" "virtio-9p-pci,fsdev=fsdev-fsRoot,mount_tag=/dev/root")
+        KERNEL_FLAGS+=("root=/dev/root" "rootfstype=9p" "init=$INIT")
+        KERNEL_FLAGS+=("rootflags=trans=virtio,version=9p2000.L,msize=5000000,cache=mmap,posixacl")
+    fi
+
+    QEMU_FLAGS+=("-fsdev" "local,security_model=none,multidevs=remap,id=fsdev-fsOutputDir,path=$OUTPUT_DIR")
+    QEMU_FLAGS+=("-device" "virtio-9p-pci,fsdev=fsdev-fsOutputDir,mount_tag=/dev/output_dir")
+
+    test -z "$KERNEL" || QEMU_FLAGS+=("-kernel" "$KERNEL")
+    test -z "$SINGLE" || KERNEL_FLAGS+=("init=/bin/sh")
 fi
 
-QEMU_FLAGS+=("-fsdev" "local,security_model=none,multidevs=remap,id=fsdev-fsOutputDir,path=$OUTPUT_DIR")
-QEMU_FLAGS+=("-device" "virtio-9p-pci,fsdev=fsdev-fsOutputDir,mount_tag=/dev/output_dir")
+if [[ "$ARCH" == "aarch64" ]] ; then
+    RAW_BOOTRAM_LOADER="-device loader,file=$BOOTRAM_IMAGE,addr=0"
+    ITB_IMAGE_LOADER="-device loader,file=$KERNEL,addr=0x10000000000"
 
-test -z "$KERNEL" || QEMU_FLAGS+=("-kernel" "$KERNEL")
-test -z "$SINGLE" || KERNEL_FLAGS+=("init=/bin/sh")
+    QEMU_FLAGS+="$RAW_BOOTRAM_LOADER $ITB_IMAGE_LOADER"
+
+    echo "KERNEL": $KERNEL
+    echo "BOOTRAM_IMAGE": $BOOTRAM_IMAGE
+    echo "QEMU_BINARY: $QEMU_BINARY"
+    echo "QEMU_FLAGS: $QEMU_FLAGS"
```

Comment:
Are these for debug?  I would remove them.

Reply:
Yeah.  Removed them.